docs/evaluate/temporal-cloud/service-availability.mdx

```diff
@@ -51,6 +53,18 @@ Currently, Temporal Cloud operates in several regions on Amazon Web Services (AW
 
 _**Your Workers and Client code aren't required to be hosted on AWS.**_
 
+### GCP
+
+Temporal Cloud operates in two regions on Google Cloud (GCP):
+
+| Area          | Code                 | Region            |
+| ------------- | -------------------- | ----------------- |
+| North America | us-west1             | Oregon            |
+| Asia Pacific  | australia-southeast1 | Sydney            |
```

Comment:
I'm very confused as to why for some we put the state, and for others we put the city 🤷‍♂️

Reply:
I hope this helps: 
https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html
https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html

Reply:
Sorry, I said "we" but I know a lot of providers do it. I think on Google documentation they do put the city though, but always feels weird we don't use the same unit of precision!